README.md

```diff
@@ -271,6 +258,4 @@ I would like to thank the reviewers who have viewed and commented on my code.
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
[OPTIONAL]
- [ ] Great job on adding the license to the project directory as requested by the previous reviewer, however, You should consider purring the correct link to this file in your README.
Hint:  You may consider saying:
```sh
This project is [MIT](./MIT.md) licensed         // since that is how your licenced file is named
```